testutils/log_utils.go

```diff
@@ -0,0 +1,22 @@
+package testutils
+
+import (
+	"testing"
+
+	"github.com/edaniels/golog"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap/zaptest"
+	"go.uber.org/zap/zaptest/observer"
+)
+
+// NewInfoObservedTestLogger is a copy of NewObservedTestLogger with info level
+// debugging instead of debug level.
+func NewInfoObservedTestLogger(tb testing.TB) (golog.Logger, *observer.ObservedLogs) {
```

Comment:
@erd I can move this to `golog`, but I wasn't sure how useful it was/whether it was even necessary.